conjur-api/User.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Conjur
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Conjur user reference
+    /// </summary>
+    /// a user represents resource for a human identity
+    public class User : Resource
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="T:Conjur.User"/> class.
+        /// </summary>
+        /// <param name="client">Client.</param>
+        /// <param name="name">Name.</param>
```

Comment:
Please make these more verbose. (OTOH these are internal, so arguably docstrings aren't that important here.)